ceph-releases/kraken/ubuntu/16.04/daemon/common_functions.sh

```diff
@@ -45,21 +45,33 @@ function create_socket_dir {
   chown ceph. /var/run/ceph
 }
 
+# create the bootstrap directories
+function create_bootstrap_directories{
+  for directory in osd mds rgw; do
+    mkdir -p /var/lib/ceph/bootstrap-$directory
```

Comment:
I only find a new occurences of mkdir -p /var/lib/ceph/osd, not any other.
Does it worth moving all of them into that function ?